build-tools/tasks/test-utils.js

```diff
@@ -18,12 +19,59 @@ const configs = {
     buildFinderInterface: ({ componentName }) =>
-      `find${componentName}(selector?: string): ${toWrapper(componentName)} | null;`,
+      `/**
+        * Returns the wrapper of the first ${componentName} that matches the specified CSS selector.
+        * If no CSS selector specified, returns the wrapper of the first ${componentName}.
+        * If no matching ${componentName} is found, returns \`null\`.
+        *
+        * @param {string} [selector] CSS Selector
+        * @returns {${toWrapper(componentName)} | null}
+        */
+       find${componentName}(selector?: string): ${toWrapper(componentName)} | null;
+       /**
+        * Returns the wrappers of all ${pluralize(componentName)} that match the specified CSS selector.
+        * If no CSS selector specified, returns all of the ${pluralize(componentName)} inside current wrapper.
+        * If no matching ${componentName} is found, returns an empty array.
+        *
+        * @returns {Array<${toWrapper(componentName)}>}
+        */
+       findAll${pluralize(componentName)}(selector?: string): Array<${toWrapper(componentName)}>;
+       /**
+        * Returns the first ${componentName} that matches the specified test id.
+        * Looks for \`data-testid\` attribute assigned to the component.
+        * If no matching ${componentName} is found, returns \`null\`.
+        *
+        * @param {string} testId
+        * @returns {${toWrapper(componentName)} | null}
+        */
+       find${componentName}ByTestId(testId: string): ${toWrapper(componentName)} | null;`,
   },
   selectors: {
     defaultExport: `export default function wrapper(root: string = 'body') { return new ElementWrapper(root); }`,
     buildFinderInterface: ({ componentName }) =>
-      `find${componentName}(selector?: string): ${toWrapper(componentName)};`,
+      `/**
+        * Returns a wrapper that matches a ${componentName} with the specified CSS selector.
+        * If no CSS selector is specified, returns a wrapper that matches ${componentName}.
+        *
+        * @param {string} [selector] CSS Selector
+        * @returns {${toWrapper(componentName)}}
+        */
+       find${componentName}(selector?: string): ${toWrapper(componentName)};
+       /**
+        * Returns a wrapper that matches all of the ${pluralize(componentName)} satisfying the specified CSS selector.
+        * If no CSS selector specified, returns a wrapper that matches all of the ${pluralize(componentName)} inside current wrapper.
+        *
+        * @returns {MultiElementWrapper<${toWrapper(componentName)}>}
+        */
+       findAll${pluralize(componentName)}(selector?: string): MultiElementWrapper<${toWrapper(componentName)}>;
+       /**
+        * Returns a wrapper that matches ${componentName} with the specified test id.
+        * Matches \`data-testid\` attribute assigned to the component.
```

Comment:
Consistency with line 41:

```suggestion
        * Looks for the  \`data-testid\` attribute assigned to the component.
```